code-challenges.test.js

```diff
@@ -52,5 +105,31 @@ var numbersToAdd2 = [0, 7, -8, 12]
 var numbersToAdd3 = []
 // Expected output: []
 
+// I totally understand the importance of pseudocode, I do however also see how the test that we use here is in itself pseudocode. i.e. ====
```

Comment:
Yup. The goal is to have your test describe expected behavior. Very similar to pseudocode.